api/src/Entity/Activity.php

```diff
@@ -22,7 +21,7 @@
 #[ApiResource(
     collectionOperations: [
         'get' => [
-            'normalization_context' => ['groups' => ['read', 'Activity:ActivityResponsibles']],
+            'normalization_context' => ['groups' => ['read', 'Activity:ActivityResponsibles', 'Activity:ScheduleEntries']],
```

Comment:
In ScheduleEntryLinks.vue https://github.com/ecamp/ecamp3/pull/2683/files#diff-88775c426170bcdeb7ef38ff0453b604a09d3f830d372c911e33286d568ac63aR5 we list/display all ScheduleEntries of an Activity.

Before this PR, a separate network request was necessary for every activity. With embedding ScheduleEntries into Activity collection, I can load all Activities and ScheduleEntries of a camp with a single network request and traverse in both sides without additional network request.

Infinite Loop should not happen with our setup of normalization groups, unless you start to explicitly embedded the return relations in the normalization context, i.e.:
```
'normalization_context' => ['groups' => ['read', 'Activity:ActivityResponsibles', 'Activity:ScheduleEntries', 'ScheduleEntry:Activity']],
```

A potential (and probably more performant?) alternative could be to avoid `RelatedCollectionLink` on `activity->scheduleEntries` (similar as for `contentNode->children`). In that case, I can load all data of a camp with 2 network request (1 for all activities + 1 for all scheduleEntries) and I can still traverse freely in both directions.